platform/test/spec/testToolConfigValidatorSpec.js

```diff
@@ -0,0 +1,84 @@
+/*global describe, it, expect, beforeEach --  functions provided by Jasmine */
+import { ToolConfigValidator } from "../../src/ToolConfigValidator"
+import { TOOL_1PANELDEF_1FUNCTION } from "../resources/TestToolFiles.js";
+import { ConfigValidationError } from "../../src/ConfigValidationError.js"
+
+
+const EXPECTED_FILE_TYPE = "ToolConfig"; 
+
+describe("ToolConfigValidator", () => {
```

Comment:
Is this really testing all error conditions that the validator checks for?

Reply:
No its schema validation, there is another validator ActivityValidator that does more of the semantic checking that has its own tests in #190.